src/main/java/net/atos/zac/zoeken/model/index/ZoekObjectType.java

```diff
@@ -5,13 +5,15 @@
 
 package net.atos.zac.zoeken.model.index;
 
-import net.atos.zac.zoeken.model.TaakZoekObject;
-import net.atos.zac.zoeken.model.ZaakZoekObject;
 import net.atos.zac.zoeken.model.ZoekObject;
+import net.atos.zac.zoeken.model.zoekobject.DocumentZoekObject;
+import net.atos.zac.zoeken.model.zoekobject.TaakZoekObject;
+import net.atos.zac.zoeken.model.zoekobject.ZaakZoekObject;
 
 public enum ZoekObjectType {
     TAAK(TaakZoekObject.class),
-    ZAAK(ZaakZoekObject.class);
+    ZAAK(ZaakZoekObject.class),
```

Comment:
Ik zou deze enum class verwijderen en overal een insrtance of doen. Zoals gezgd kun je in Java 17 in een case of in instance meteen toekennen aan een variabele